Tests/Test_Minions.c

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+#include "Minions.h"
+int main(void){
+    Minion_Init();
+    BSP_UART_Init(UART_2);
+    Minion_Error_t e;
+
+    while(1){
+        printf("IGN_1: %d\n\r", Minion_Read_Pin(IGN_1, &e));
+        printf("IGN_2: %d\n\r", Minion_Read_Pin(IGN_2, &e));
+
+        for(int i =0; i < 999999; i++){}
```

Comment:
This is a fine test file to use to make sure ignition stuff is working, but I would either rename the test file to be more descriptive of what's actually in it or actually flesh out the test file to test all minions, or remove it from the PR.